### PR TITLE
[signing] Document the HSM initialization ceremony

### PR DESCRIPTION
**Pull request by cfrantz**
This PR belatedly documents the HSM initialization and key generation process for `earlgrey_a0`.

### CODE REVIEW

File: signing/logs/2023-06-06.md [hunk 1]
```diff
@@ -0,0 +1,636 @@
...
+]
+```
+
+### Generate Test Signatures
+
+The test signatures were copied into `//sw/device/silicon_creator/rom/keys/real/rsa`.
+See the documentation in that directory for details about how the test signatures are used in the unit tests.
+```bash
+(subshell)$ echo -n "test" >test.txt
+(subshell)$ hsmtool --logging=info --token earlgrey_a0 -u user exec earlgrey_test_signatures.hjson
```
Comment (moidx) [inline signing/logs/2023-06-06.md:580]: Should all `hsmtool` config files be archived with the logs? Or alternatively, should `hstmtool` print the command in the result log?

**Review by moidx — APPROVED** [review]

File: signing/logs/2023-06-12.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+Since moidx@opentitan.org was not present for HSM initialization, he did not have his HSM credentials.
+
+cfrantz opened KK2917761 (ACS cards) and KK2917762 (Nitrokeys):
+- Delivered ACS#1 and OCS#1 to ttrippel.
+- cfrantz took possession of ACS#2 and OCS#2.
+- Sealed ACS#3, OCS#3 and NitroKey `earlgrey_a0_002` into evidence bag KK2917767 for moidx.
+- Resealed ACS 4-8 into evidence bag KK2917765.
+- Resealed NitroKeys 3-4 into evidence bag KK2917766.
+- Sealed NitroKeys 6-7 into evidence bag KK2917760.
```
Comment (timothytrippel) [inline signing/logs/2023-06-12.md:21]: I think this should be keys 5-6

File: signing/logs/2023-06-06.md (hunk 1 above)
Comment (cfrantz) [reply signing/logs/2023-06-06.md:580]: The config templates for keygen and export are already committed to the codebase under `//sw/device/silicon_creator/rom/keys/real/rsa/...`.  I'll add links from this document to those files so people reading the logs can easily find the corresponding inputs.

**Comment by moidx** [comment]
nit: Since the logs were committed before branching `earlgrey_es`, should we cherrypick into that branch for archival purposes?

**Comment by cfrantz** [comment]
> nit: Since the logs were committed before branching `earlgrey_es`, should we cherrypick into that branch for archival purposes?

Yes.

QQ: the logs use email addresses to refer to various individuals (mostly opentitan.org emails, but one google.com email).  Should I keep them as email addresses, or should I remove the domain names to prevent spam bots from scraping the addresses?

**Comment by moidx** [comment]
> QQ: the logs use email addresses to refer to various individuals (mostly opentitan.org emails, but one google.com email). Should I keep them as email addresses, or should I remove the domain names to prevent spam bots from scraping the addresses?

Good point. We can use GitHub or OpenTitan usernames. I am inclined towards using OpenTitan usernames as that ties into the OpenTitan organizational processes, which helps wrt auditing. GH accounts are similar but with an additional level of indirection.

**Comment by cfrantz** [comment]
> Good point. We can use GitHub or OpenTitan usernames. I am inclined towards using OpenTitan usernames as that ties into the OpenTitan organizational processes, which helps wrt auditing.

Ok, I removed the full email addresses in favor of OT usernames.

**Pull request merged**